jobs/cloud_controller_ng/templates/cloud_controller_ng.yml.erb

```diff
@@ -255,6 +255,12 @@ uaa:
   <% if_p("uaa.cc.token_secret2") do |token_secret2| %>
   symmetric_secret2: "<%= token_secret2 %>"
   <% end %>
+  clients:
```

Comment:
This can lead to a failure in the pre-start action if no `uaa.clients.cloud_controller_shadow_user_creation.secret` is configured:
```
[2024-12-03 08:01:19+0000] Membrane::SchemaValidationError: { uaa => { clients => Expected instance of Array, given instance of NilClass } } (Membrane::SchemaValidationError)
```

Reply:
With the patch it works now.